packages/start-frontend/src/helpers/prompt-clack.js

```diff
@@ -0,0 +1,128 @@
+// to execute this file
+// run node ./packages/start-frontend/src/helpers/prompt-clack.js
+
+import {
+  intro,
+  outro,
+  spinner,
+  text,
+  select,
+  group,
+  multiselect,
+  confirm,
+} from "@clack/prompts";
+import color from "picocolors";
+import { setTimeout as sleep } from "node:timers/promises";
+
+const CLIOptions = {
+  react: {
+    name: "React",
+    apiSolution: [
+      { name: "RESTful", value: "restful" },
+      { name: "GraphQL", value: "graphql" },
+    ],
+    useModules: [{ name: "Add CRUD Operations", value: "crud" }],
+  },
+};
+
+async function promptClack() {
+  // intro example
+  intro(color.blue(`create some app with Clack!`));
+
+  // group example
+  const groupUtility = await group(
+    {
+      location: () =>
+        // text example
+        text({
+          message: color.blue("Which location you want to start project?"),
+          placeholder: "./my-app",
+          validate: (value) => {
+            if (!value) {
+              return "Please provide a location path";
+            }
```

Comment:
Yes, but then user has to remove this default value every time when they generate package in their own path.
Personally, I don't think we need it since I believe most of user should name their own project name instead of default.